src/stories/components/RecipeReviews/RecipeReviews.js

```diff
@@ -17,58 +22,166 @@ import { FormItem } from '../FormItem/FormItem';
  * )
  */
 export const RecipeReviews = ({ recipeId }) => {
+  // to leave username as part of review
+  const { currentUser } = useContext(AuthContext);
+  const [profile, setProfile] = useState(null);
+  const history = useHistory();
+
+  const [errorMessage, setErrorMessage] = useState('');
+  const [successMessage, setSuccessMessage] = useState('');
   const [reviews, setReviews] = useState(null);
   const [review, setReview] = useState({
-    recipeId,
+    // recipeId,
     title: '',
     content: '',
     rating: '',
-    date: (new Date).toLocaleDateString('default', {
-      month: 'long',
-      day: 'numeric',
-      year: 'numeric',
-    }),
-    user: 'apizzoccheri',
+    username: '',
+    date: UTILS.convertDateToString(new Date()),
   });
 
   useEffect(() => {
-    // TODO make call to DB to fecth the recipe's reviews
-    setReviews([{
-      title: 'Test',
-      content: 'Hello world',
-      rating: '4',
-      username: 'apizzoccheri',
-      date: new Date(),
-    }]);
-  }, [reviews]);
-
-  // push review to database
+    setErrorMessage('');
+    setSuccessMessage('');
+    // get profiles username to attach to reviews
+    if (currentUser) {
+      database.getProfile(currentUser.uid)
+        .then((doc) => {
+          setProfile(doc.data());
+        })
+        .catch((error) => {
+          // setError(error)
+        });
+    }
+
+    // this call is purely to make document discoverable in future
+    database.setRecipe(recipeId);
+    // this one actually gets data
+    setReviewsFromDB();
+  }, [currentUser]);
+
+
+  /**
+   * Makes call to Firestore, grabs all reviews for this recipe
+   *   and saves them to the local state variable 'reviews'
+   */
+  const setReviewsFromDB = () => {
+    setReviews([]);
+    database.getReviewsForRecipe(recipeId)
+      .then((collection) => {
+        const theReviews = [];
+        collection.forEach((doc) => {
+          theReviews.push(doc.data());
+        });
+        setReviews(theReviews);
+      });
+  };
```

Comment:
Re: Mothil's comments, is this comment out of date now? 

This will keep errors from the promise, its just a forEach to get each individual review from the promise returned from the ```recipeService```'s ```readReviewsForRecipe()``` method